docs/build_from_source.md

```diff
@@ -172,7 +147,51 @@ When set to `0`, OpenVINO&trade Model Server will be built with [MediaPipe](medi
 
 Example:
 ```bash
-make docker_build MEDIAPIPE_DISABLE=0
+make release_image MEDIAPIPE_DISABLE=1
+```
+
+### `PYTHON_DISABLE`
+
+When set to `0`, OpenVINO&trade Model Server will be built with [Python Nodes](python_support/quickstart.md) support. Default value: `0`. 
+
+Example:
+```bash
+make release_image PYTHON_DISABLE=1
+```
+
```

Comment:
Maybe add note that Python is enabled via MediaPipe and configuration with `MEDIAPIPE_DISABLE=1 PYTHON_DISABLE=0` is invalid